src/test/common/clara/test_accumulation.cljc

```diff
@@ -69,6 +69,16 @@
   [& args]
   (apply = args))
 
+;; In ClojureScript the type function does not use the :type key in metadata as in Clojure, but we had
+;; tests in Clojure that were relying on this.  Creating a custom function for the fact type along these
+;; lines was the easiest way to get the tests relying on this behavior to pass in ClojureScript.  I used
+;; data on the actual fact rather than metadata and updated the tests accordingly.  This is desirable since
+;; Clara's value-based semantics won't consider the metadata part of the fact for the purpose of memory operations.
```

Comment:
See https://github.com/cerner/clara-rules/pull/332#issuecomment-320998627 for related to this.